app/spec/services/github_checks_verifier_spec.rb

```diff
@@ -139,6 +139,18 @@
       service.send(:apply_filters, checks)
       expect(checks.map(&:name)).to all(eq "check_name")
     end
+    
+    it "filters out only ignore_checks" do
+      checks = [
+        OpenStruct.new(name: "check_name1", status: "queued"),
+        OpenStruct.new(name: "check_name2", status: "queued"),
+        OpenStruct.new(name: "other_check", status: "queued")
+      ]
+
+      service.config.ignore_checks = ["check_name1,check_name2"]
```

Comment:
This test fails when I run it locally, but removing the [ ] fixes it:

```suggestion
      service.config.ignore_checks = "check_name1,check_name2"
```

Reply:
@omkarkhatavkar also thanks for the PR!  Great feature.

Reply:
@hwood-fg not much used ruby recently, chat-gpt helped in between